### PR TITLE
Add support for Canal+ TV group channels

### PR DESCRIPTION
**Pull request by melmorabity**
This plugin adds support for French Canal+ TV group channels:
* canalplus.fr
* c8.fr
* cstar.fr

Both live and VOD are supported.

### CODE REVIEW

File: docs/plugin_matrix.rst [hunk 1]
```diff
@@ -37,6 +37,9 @@ bnt                 - tv.bnt.bg          Yes   No    Streams may be geo-restrict
                     - nova.bg
 bongacams           bongacams.com        Yes   No    Only RTMP streams are available.
 btv                 btv.bg               Yes   No    Requires login, and geo-restricted to Bulgaria.
+canaplus            - canalplus.fr       Yes   Yes   Streams may be geo-restricted to France.
+                    - c8.fr
+		    - cstar.fr
```
Comment (beardypig) [inline docs/plugin_matrix.rst]: Is `- cstar.fr` misaligned?

**Comment by melmorabity** [comment]
Good catch, I've just fixed alignment.

**Comment by beardypig** [comment]
Thanks @melmorabity :)

**Comment by campones** [comment]
sorry but how am I supposed to use this plugin?  what is the command line please

**Comment by campones** [comment]
Hi 

after spending few hours guessing  I could find the url for  cstar and c8, since I don't have iphone, as been a "wild" guess we will say..  on android they use hss streaming so that's not an option, and I know they use hls for iOS but without iphone, hard to sniff anyway

https://hls-m002.live-lv3.canalplus-cdn.net/live/disk/c8-hd/ios-hd/index.m3u8

so my issue is this is encrypted and I was wondering if there was a work around for this?. 

Also as I said in august, I still didn't get how to use the canal.py plugin, what is the url I m supposed to enter?

**Comment by bastimeyer** [comment]
You are commenting on a pull request that got merged almost two years ago. If you have an issue or a question, please open a new thread on the issue tracker.

**Pull request merged**